msg/servo_report.msg

```diff
@@ -0,0 +1,8 @@
+uint64 timestamp					# time since system start (microseconds)
+
+float32 servo_position					# feedback position of servo
+float32 servo_speed					# feedback speed of servo
+float32 servo_force					# force experienced by servo
```

Comment:
Units?
Set to NAN if force not provided?

Reply:
Added units in latest commit. servo_force has been updated to servo_acceleration.